src/mechanics/damage.py

```diff
@@ -0,0 +1,66 @@
+class Fighter:
+    def __init__(self, life=100, max_power=40):
+        self.life = life
+        self.maxLPA = 0.10 * self.life
+        self.maxHPA = 0.18 * self.life
+        self.shieldLPA = 0.05 * self.maxHPA
+        self.shieldHPA = 0.20 * self.maxHPA

```

Comment:
@nichitatabanschi Parametrize these coefficients. You can put them as parameters with default values.